app/policies/user_policy.rb

```diff
@@ -48,7 +48,8 @@ def moderation_routes?
   end
 
   def permitted_attributes
-    %i[available_for
+    %i[allows_article_edits
```

Comment:
I think this might make more sense under /settings/misc.

Reply:
that makes sense!